airflow/opt/providers/etna/etna/etls/metis_linker.py

```diff
@@ -93,29 +93,35 @@ def data_frame_update(self, model_name, script, tail, update, metis, model):
                 if script['format']=="csv":
                     separator = ","
                 if script['format']=="tsv":
-                    separator = "\\t"
+                    separator = "\t"
                 if script['format']=="auto-detect":
                     if re.search("csv$", file.file_path) is not None:
                         separator = ","
                     elif re.search("tsv$", file.file_path) is not None:
-                        separator = "\\t"
+                        separator = "\t"
                     else:
                         # Attempt to use pandas' automated detection
                         separator = None
-                # Maybe Future Feature: expose additional blanker value in config ui
-                data = pandas.read_table(file_reader, sep=separator, engine = 'python').replace({numpy.nan: None})
+                # Load dataframe
+                data = pandas.read_table(file_reader, sep=separator, engine = 'python', na_filter=False)
             if len(data.columns) < 2:
                 raise MetisLoaderError(f"{file.file_name} seems to have fewer than 2 columns. Check the 'format' configuration for this data_frame loader.")
             if not set(columns).issubset(data.columns):
                 missing = ', '.join([col for col in columns if col not in data.columns])
                 raise MetisLoaderError(f"{file.file_name} is missing column(s) targetted by {model_name} data_frame loader 'column_map': {missing}.")
+            if pandas.isna(data).values.any():
+                raise MetisLoaderError(f"{file.file_name} has unexpected NA values after all parsing. Data rows may be shorter than the column row indicates.")
```

Comment:
Encountered a couple other errors during testing:
- Discovered from an unfortunately mis-formatted target file that `pandas.read_table()` does not error if your data rows are shorter than your header.  Instead, it fills the missing columns with `numpy.NaN`.  So **added a check and error message for presence of na values**.